modules/placement-driver/src/integrationTest/java/org/apache/ignite/internal/placementdriver/ActiveActorTest.java

```diff
@@ -99,53 +62,43 @@
 /**
  * Placement driver active actor test.
  */
-@ExtendWith({MockitoExtension.class, ConfigurationExtension.class})
+@ExtendWith(MockitoExtension.class)
 @MockitoSettings(strictness = Strictness.LENIENT)
-public class ActiveActorTest extends IgniteAbstractTest {
-    /** RAFT message factory. */
-    private static final RaftMessagesFactory FACTORY = new RaftMessagesFactory();
-
-    /** Base node port. */
-    private static final int PORT_BASE = 1234;
-
-    @InjectConfiguration
-    protected RaftConfiguration raftConfiguration;
-
-    /** RPC executor. */
-    protected ScheduledExecutorService executor = new ScheduledThreadPoolExecutor(20, new NamedThreadFactory("Raft-Group-Client", log));
-
+public class ActiveActorTest extends AbstractTopologyAwareGroupServiceTest {
     private Map<String, PlacementDriverManager> placementDriverManagers = new HashMap<>();
 
     @Mock
     MetaStorageManager msm;
 
+    @BeforeEach
+    public void setUp() {
+        when(msm.recoveryFinishedFuture()).thenReturn(completedFuture(0L));
```

Comment:
please check if it is correct to return 0 here

Reply:
it doesn't matter, it doesn't start the node and perform the actual recovery